app/models/sdg/local_target.rb

```diff
@@ -0,0 +1,36 @@
+class SDG::LocalTarget < ApplicationRecord
```

Comment:
Does it make sense to add this model (and tests) to the `SDG::Relatable` concern?

Reply:
Done. Thanks